doc/common/roadmap.md

```diff
@@ -2,38 +2,36 @@
 # Roadmap
 
 ## Table of Contents
- * [Overview](#6.1)
- * [Roadmap](#6.2.2)
-   * [High Level Roadmap](#6.2.1)
-   * [Detailed Roadmap](#6.2.2)
-   * [Detailed Milestones](#6.2.3)
- * [Dependencies between various Workstreams ](#6.3)
- * [Dependencies with Industry Communities](#6.4)
-   * [Dependencies with OPNFV and OVP](#6.4.1)
-   * [Dependencies with CNCF and OVP 2.0](#6.4.1)
-
-<a name="1"></a>
+ * [Overview]
+ * [Roadmap]
+   * [High Level Roadmap]
+   * [Detailed Roadmap]
+   * [Detailed Milestones]
+ * [Dependencies between various Workstreams ]
+ * [Dependencies with Industry Communities]
+
+
 ## Overview
-* The activities of the CNTT community are articulated around [Projects](#project), [Milestones](#2.3) and [Releases](#release).
-* CNTT is embracing simultaneous delivery model, meaning that all contributing projects have to follow the cadence and intermediate milestones.
-* Each CNTT release is the only delivery vehicle and is common for all projects.
-* The CNTT current release plan is available [here](./release_notes/release_process.md).
+* The activities of the Anuket Project community are articulated around [Workstreams (WS)](#workstream), [Milestones](#2.3) and [Releases](#release).
+* The Anuket Project is embracing simultaneous delivery model, meaning that all contributing projects have to follow the cadence and intermediate milestones.
+* Each Anuket Project release is the only delivery vehicle and is common for all projects.
+* The Anuket Project current release plan is available [here](https://wiki.anuket.io/display/HOME/Releases).
 
 
 <p align="center"><img src="./figures/Release Lifecycle.png" alt="scope" title="Milestones" width="100%"/></p>
 <p align="center"><b>Figure 1:</b> Milestones</p>
 
 **Definitions**
 
-<a name="project"></a>
```

Comment:
Why keep this anchor and rename it to workstream despite linking to the definition of "project"?

Reply:
@gkunz Line 15 refers to this portion -- "... are articulated around [Workstreams (WS)](#workstream)"

Reply:
@pgoyal01 thanks for the pointer to line 15. Still, I thought we agreed on using the term project instead of workstream. In any case, it is a bit inconsistent now as line 15 refers to workstreams, but the defintion further down talks about projects.

Reply:
@gkunz We also have WS in other parts of the documents. Let me see how we can incorporate both concepts. Thanks.

Reply:
@pgoyal01 thank you. looks good to me.